src/editors/data/services/cms/types.ts

```diff
@@ -68,9 +70,8 @@ export const problemDataProps = {
     }),
   }),
 };
+*/
 
 export default {
-  videoDataProps,
-  problemDataProps,
```

Comment:
These weren't used anymore (?), but they look like they'll be useful if/when we add more types to these APIs / to the redux state, so I kept the info here, just commented out.